alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -54,12 +55,20 @@ import org.opensearch.test.junit.annotations.TestLogging
 @Suppress("UNCHECKED_CAST")
 class SecureMonitorRestApiIT : AlertingRestTestCase() {
 
+    companion object {
+
+        @BeforeClass
+        @JvmStatic fun setup() {
+            // things to execute once and keep around for the class
+            org.junit.Assume.assumeTrue(System.getProperty("security", "false")!!.toBoolean())
```

Comment:
Do we need separate test class for scenarios with Security Disabled? Such as  ``Forbidden`` as response in one of the test below.

Reply:
That would need a complete suite written for scenarios with security plugin uninstalled. I will open an issue for the same.

Reply:
Sure, makes sense. Lets add an issue, and we can tag it with the PR.